src/Lobby.ts

```diff
@@ -0,0 +1,67 @@
+import { Player } from './Player'
+import { Loiterer } from './Loiterer'
+import { Broadcaster } from './Broadcaster'
+import { Team } from './constants/Constants'
+import { GameUtility as gu } from './GameUtility'
+import { RuleEnforcer as re } from './RuleEnforcer'
+
+import ws = require('ws')
+import * as _ from 'lodash'
+
+export class Lobby {
```

Comment:
Considering that this is a new class that folks won't be familiar with, can you add a block comment explaining what the Manager is responsible for?